pkgs/development/python-modules/seafobj/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, fetchpatch, fetchFromGitHub, python3Packages }:
```

Comment:
```suggestion
{ lib
, fetchFromGitHub
, buildPythonPackage
}:
```

Reply:
No `python3Packages`!!
Otherwise python39.pkgs.sefobj and python310.pkgs.seafobj would be the same.